### ISSUE TITLE
fatal: reference is not a tree: 'e019648e..' Unknown reference for submodule?

### ISSUE THREAD

**Issue by montdidier** [RusticiSoftware/SCORMCloud_MoodleModule#32]
It appears that the SCORMCloud_PHPLibrary submodule is incorrect. A checkin of a reference that was not itself committed to the submodule?

**Comment by danmarsden** [comment]
yes - it's a checkin that includes my pull request to improve support to php 5.5 - that pull request hasn't been accepted by Rustici yet. If it's causing issues for you, grab the zip from the moodle.org plugins db as this includes the correct version of the PHPLib with my latest patches.
